docker-compose-dev.yml

```diff
@@ -0,0 +1,35 @@
+version: '2'
+services:
+
+
+  postgres_data:
+    image: busybox
+    volumes:
+      - /var/lib/postgresql/data
+
+  postgres:
+    image: postgres:9.6
+    logging:
+      driver: "none"
```

Comment:
What is this doing?

Reply:
The `postgres_data` is an empty [volume container](https://docs.docker.com/engine/admin/volumes/volumes/) to ensure that Postgres data won't be lost when Docker restarts.

Reply:
I meant the logging thingy

Reply:
suppresses logging to the console. however, since there will be no ruby container, we might as well let the logs show. i'll update the compose file